src/backend/common/util.hpp

```diff
@@ -15,3 +15,7 @@
 #pragma once
 
 std::string getEnvVar(const std::string &key);
+
+// Dump the kernel sources only if the environment variable is defined
+static const char* saveJitKernelsEnvVarName = "AF_DUMP_JIT_KERNELS_DIR";
```

Comment:
Can you rename this to `AF_JIT_KERNEL_TRACE`?

It accepts `1` and writes it to stdout or "directory:..." and it will create a file for each kernel. In the future I will be creating a graphviz images so I want this to be slightly more flexible.

Reply:
Renamed. Now accepts stdout/stderr.